cityofaustin/atd-product
# 10616 projects page

Fixes cityofaustin/atd-data-tech/issues/10616

Steps to test:

[Netlify link](https://deploy-preview-52--atd-product.netlify.app/)

1. Navigate to the Projects page
2. Click on all of the status filters, make sure they look and function as expected.
3. Repeat for the workgroup filter.
4. Toggle between List/Chart view, make sure this works as expected.
5. Scale down your screen to the next break point (tablet), compare the layout to the mock up in the [issue](https://github.com/cityofaustin/atd-data-tech/issues/10616), make sure everything functions
6. Repeat for the next breakpoint (mobile)

One thought I have: The new design changes the workgroup dropdown from saying "Any workgroup" to "By workgroup" by default. If you click on the drop down, select a workgroup, then want to go back to displaying all work groups, you would have to click "By workgroup" again, which I don't think is as obvious as "Any workgroup". Curious to know what you all think!
